driver/UFSDriver.F90

```diff
@@ -367,6 +370,14 @@ subroutine SetModelServices(driver, rc)
             found_comp = .true.
           end if
 #endif
+#ifdef FRONT_MPAS
+!          if (trim(model) == "mpas") then
+!            call NUOPC_DriverAddComp(driver, trim(prefix), MPAS_SS, &
+!              info=info, petList=petList, comp=comp, rc=rc)
+!            if (ChkErr(rc,__LINE__,u_FILE_u)) return
+!            found_comp = .true.
+!          end if
+#endif
```

Comment:
Is there a reason this is commented out currently?

Reply:
It's just a placeholder for the timebeing. 
The NUOPC pieces for MPAS will be added later, when the underlying atmospheric component work on MPAS is further along.